notes/data_cleaning.md

```diff
@@ -74,6 +74,51 @@ df.fillna(method='ffill', axis=1)
 df.fillna(method='ffill', limit=1)
 ```
 
+### Make columns python friendly
```

Comment:
the whole part in this file with the definition seems redundante to me. with the new name 'clean_column_names' it is very self explanatory and I think the example code is enough.

Reply:
Feel free to delete redundant information.

Reply:
same as above. I do not have the time to edit the code of 11 people.